src/test/java/com/microsoft/sqlserver/jdbc/resultset/ResultSetTest.java

```diff
@@ -94,13 +95,14 @@ public void cleanUp() throws Exception {
      */
     @Test
     @Tag(Constants.xAzureSQLDW)
+    @Tag(Constants.JSONTest)
```

Comment:
This concerns me a bit. I assume we are generally excluding JSON tests initially, until we stand up some new servers/images that support it in test pipelines. This effectively turns this test off for all the non-JSON stuff in it, too, until that happens. It would be better to split JSON into its own test here.